pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPool.sol

```diff
@@ -77,6 +73,6 @@ contract SiloLinearPool is LinearPool, Version {
     }
 
     function _getWrappedTokenRate() internal view override returns (uint256) {
-        return _exchangeRateModel.calculateExchangeValue(_shareToken);
+        return _calculateExchangeValue(_shareToken);
```

Comment:
Are we absolutely certain we don't need to apply any decimal scaling here? Will this always return an 18-decimal rate regardless of main/wrapped decimals?